src/ol/map.js

```diff
@@ -1490,6 +1490,8 @@ ol.Map.createOptionsInternal = function(options) {
     var logo = options.logo;
     if (typeof logo === 'string') {
       logos[logo] = '';
+    } else if (goog.dom.isElement(logo)) {
```

Comment:
I tried doing it but I got an error while building, it told me the check was always going to be false since logo was always going to be a string. I probably missed something, I'll continue checking tomorrow.